CONTRIBUTING.md

```diff
@@ -0,0 +1,163 @@
+    ```
+
+2. Push your topic branch up to your fork:
+
+    ```sh
+    git push origin <topic-branch-name>
+    ```
+
+3. [Open a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) with a clear title and a detailed description explaining the reasons for the changes. Make sure there is sufficient information for the reviewer to understand your changes.
+
+4. Check if the CI/CD pipelines have passed. Address the errors if they have not.
+
+> [!IMPORTANT]  
+> By submitting a patch, you agree to license your work under the same license as that used by the project.
+
+
+### Everyone can contribute
+
+An Open Source project such as this one has a wide range of needs, and there are a lot of opportunities to help:
+
+- You can expand or improve the test suite
+- You can improve the various documentation resources in general, just picking something to expand upon, edit or check for typos
+- You can evangelize for the project: present or talk about Sequoia PGP in your company, local tech meetup, youtube channel, or similar circles
+
+
+### I have submitted my Pull Request, what are the next steps?
+
+First of all, 🙏 thank you for your contribution! Sit back and relax.
+
+As soon as possible, usually within a few weeks, a team member will review your pull request and provide comments.
+
+
+## Reporting bugs
+
+A bug is a _demonstrable problem_ that is caused by the code in our repository. Good bug reports are extremely helpful!
+
+A well-written bug report shouldn’t require others to contact you directly to obtain additional details. Kindly ensure that your report contains as much detail as you can. What surrounds you? In what ways can the problem be replicated? Which operating system is affected by the issue? What result would you anticipate? All of these specifics will aid in the correction of any possible bugs.
+
+Before you create a bug report, you should:
+
+1. **Use the GitHub issue search** - check if the issue has already been reported.
+
+2. **Check if the issue has been fixed** - try to reproduce it using the latest `main` branch in the repository.
+
+3. **Isolate the problem** - ideally create a reduced test case.
+
+To create a new bug report, add a new [issue](https://github.com/cabotage/cabotage-app/issues).
+
+
+## Disclose a security vulnerability
+
+If you've identified a security issue, please do not report it publicly on GitHub issues or any other public form. Instead, navigate to GitHub's Security tab and click on Report a vulnerability. Fill out [the form](https://github.com/cabotage/cabotage-app/security/advisories/new), providing as much relevant information as possible, including steps to reproduce the issue. Thank you for helping us keep our project secure!
+
+
+## Get in touch
+
+Please feel free to reach out to us on our [Github discussions](https://github.com/cabotage/cabotage-app/discussions)
+
```

Comment:
There aren't any discussions. Shall we maybe ask before we add this section?

Reply:
They aren't any yet because they activated it yesterday. I asked them how they wanted to be contacted and GH discussions was agreed